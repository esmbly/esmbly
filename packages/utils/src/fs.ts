import fs from 'fs';
import util from 'util';

export default {
  readdir: util.promisify(fs.readdir),
  readFile: util.promisify(fs.readFile),
  stat: util.promisify(fs.stat),
  writeFile: util.promisify(fs.writeFile),
};
