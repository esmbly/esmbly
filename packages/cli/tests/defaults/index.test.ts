import fs from 'fs';
import path from 'path';

describe('Default configuration', () => {
  it('exposes the correct default configuration', () => {
    const configPath = path.join(__dirname, '../../defaults/.esmblyrc.js');
    const content = fs.readFileSync(configPath, 'utf8');
    expect(content).toMatchSnapshot();
  });
});
