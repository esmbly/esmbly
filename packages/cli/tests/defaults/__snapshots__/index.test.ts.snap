// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default configuration exposes the correct default configuration 1`] = `
"module.exports = {
  files: [],
  transformers: [],
  output: []
}
"
`;
